fendermint/app/src/cmd/genesis.rs

```diff
@@ -94,6 +95,8 @@ cmd! {
             set_ipc_gateway(&genesis_file, args),
         GenesisIpcCommands::FromParent(args) =>
             new_genesis_from_parent(&genesis_file, args).await,
+        GenesisIpcCommands::SealState(args) =>
```

Comment:
I wonder if it makes sense to have this under IPC?

Reply:
Currently the separation btw ipc and non-ipc is blurred. I would actually argue we track this in another issue, because it's the downstream implementation that assumes an `artifacts` folder which makes it difficult to split upstream.